python_modules/dagster/dagster_tests/launcher_tests/test_default_run_launcher.py

```diff
@@ -116,24 +116,25 @@ def math_diamond():
 @repository
 def nope():
     return [
-        noop_pipeline,
-        crashy_pipeline,
-        exity_pipeline,
-        sleepy_pipeline,
-        slow_pipeline,
+        noop_job,
+        crashy_job,
+        exity_job,
+        sleepy_job,
+        slow_job,
         math_diamond,
     ]
 
 
 def run_configs():
     return [
         None,
-        {"execution": {"multiprocess": {}}},
+        {"execution": {"config": {"in_process": {}}}},
     ]
 
 
-def _is_multiprocess(run_config):
-    return run_config and "execution" in run_config and "multiprocess" in run_config["execution"]
+# jobs default to multiprocess without additional config
+def _is_multiprocess(run_config: Optional[Mapping[str, Any]]) -> bool:
+    return run_config is None
```

Comment:
this isn't really correct right? Possible to have config and it's still multiprocess

Reply:
good point, I just deleted this and inlined the `run_config is None` check with a comment